src/chrome/content/rules/openscad.org.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	OpenSCAD
+	Open source 3D design tool
+
+	Refused:
+		openscad.org
```

Comment:
Can be redirected/secured to `www`

Reply:
I do not know how to redirect from `http://openscad.org` to `https://www.openscad.org` by a ruleset.

Reply:
Just add `<rule from="^http://openscad\.org/" to="https://www.openscad.org/"/>` before the `<rule from="^http:" to="https:" />` rule and add `openscad.org` as target.